_workinggroups/softwaredevelopertools.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Software/Developer Tools 
+layout: plain
+---
+
+The Software/Developer Tools Working Group aims to help the HEP community in developing the latest, modern feature-rich projects
+with the aid of software tools such as code editors, static/dynamic code analyzers, compilers, debuggers, performance analyzers and so on.
+The group plans to accomplish this by mainly serving as a forum where developers can share their experiences, discuss new/existing tools they utilize,
+and learn from each other. To that point, contributions from the entire community are strongly encouraged.
+
+The group is currently coordinated by Giulio Eulisse (CERN, EP-AIP-SDS), Alaettin Serhan Mete (UC Irvine, ATLAS), and Martin Ritter (LMU, Belle II).
+The Indico pages for the working group meetings can be found at <https://indico.cern.ch/category/10415/> and
+all coordinators can be reached at <XYZ@googlegroups.com>.
```

Comment:
Just the addition of the tech list for group discussions missing now.

Reply:
I create a new group and update this link.

Reply:
Oh, will do that too, missed it :)

Reply:
OK done.